x-pack/plugins/enterprise_search/public/applications/app_search/components/settings/log_retention/log_retention_confirmation_modal.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiTextColor, EuiOverlayMask } from '@elastic/eui';
+import { useActions, useValues } from 'kea';
+
+import { GenericConfirmationModal } from './generic_confirmation_modal';
+import { LogRetentionLogic } from './log_retention_logic';
+
+import { ELogRetentionOptions } from './types';
+
+export const LogRetentionConfirmationModal: React.FC = () => {
+  const CANNOT_BE_RECOVERED = i18n.translate(
+    'xpack.enterpriseSearch.appSearch.settings.logRetention.modal.recovery',
+    {
+      defaultMessage: 'cannot be recovered',
+    }
+  );
```

Comment:
I'm slightly concerned about this from a i18n standpoint - I'm not sure grammatically we can just pull out a clause like this and expect our translation team to understand the context and have it make sense grammatically in other languages like Chinese and Japanese.

That being said I'm not sure I know of a better way myself - let me try and check with the localization team.

Reply:
Oh right, that's a good point. Like, I wonder how generally you include html like this in i18n strings.

Reply:
For the sake of this PR I'm just going to change it to this:
![screencapture-localhost-5601-app-enterprise-search-app-search-settings-account-2020-11-11-09_02_29](https://user-images.githubusercontent.com/1427475/98820829-b62c0100-23fc-11eb-96f5-2f5c47c0417d.png)

```
            <>
              <p>
                {i18n.translate(
                  'xpack.enterpriseSearch.appSearch.settings.logRetention.modal.api.description',
                  {
                    defaultMessage:
                      'When disabling API Logs, all your engines will immediately stop indexing API Logs. Your existing data will be deleted in accordance with the storage timeframes outlined above.',
                  }
                )}
              </p>
              <p>
                <strong>
                  <EuiTextColor color="danger">{CANNOT_BE_RECOVERED_TEXT}</EuiTextColor>
                </strong>
              </p>
            </>
```

Reply:
https://github.com/elastic/kibana/pull/83009/commits/2737c20477dd8494108e2a39ca1089a2dbcc7325

Reply:
This is an awesome solution, thanks Jason! I have a conversation open w/ the Kibana localization channel now asking them what their recommendation would be for the original scenario - will update if I hear back!